src/test/java/step1/StringCalculatorTest.java

```diff
@@ -0,0 +1,40 @@
+package step1;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static step1.StringCalculator.calculate;
+
+public class StringCalculatorTest {
```

Comment:
```suggestion
class StringCalculatorTest {
```
junit 5부터는 테스트 클래스에 pulibc을 제거해주셔도 됩니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods